README.md

```diff
@@ -1,33 +1,65 @@
-# KlaviyoSwift
+# klaviyo-swift-sdk
 
-[![CI Status](https://travis-ci.org/klaviyo/klaviyo-swift-sdk.svg?branch=master)](https://travis-ci.org/klaviyo/klaviyo-swift-sdk)
-[![Swift](https://img.shields.io/badge/Swift-5.6_5.7-orange?style=flat-square)](https://img.shields.io/badge/Swift-5.6_5.7-Orange?style=flat-square)
+![CI status](https://github.com/klaviyo/klaviyo-swift-sdk/actions/workflows/swift.yml/badge.svg)
 [![Swift Package Manager](https://img.shields.io/badge/Swift_Package_Manager-compatible-orange?style=flat-square)](https://img.shields.io/badge/Swift_Package_Manager-compatible-orange?style=flat-square)
+![SPM version](https://img.shields.io/github/v/release/klaviyo/klaviyo-swift-sdk)
 [![Version](https://img.shields.io/cocoapods/v/KlaviyoSwift.svg?style=flat)](http://cocoapods.org/pods/KlaviyoSwift)
 [![License](https://img.shields.io/cocoapods/l/KlaviyoSwift.svg?style=flat)](http://cocoapods.org/pods/KlaviyoSwift)
-[![Platform](https://img.shields.io/cocoapods/p/KlaviyoSwift.svg?style=flat)](http://cocoapods.org/pods/KlaviyoSwift)
+![Minimum deployment version](https://img.shields.io/badge/minimum_iOS_deployment_target-iOS13-brightgreen)
+
+## Contents
+- [Overview](#overview)
+- [Installation](#installation)
+- [Initialization](#initialization)
+- [Profile Identification](#profile-identification)
+  - [Reset Profile](#reset-profile)
+  - [Anonymous Tracking Notice](#anonymous-tracking-notice)
+- [Event tracking](#event-tracking)
+- [Push Notifications](#push-notifications)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Collecting Push Tokens](#collecting-push-tokens)
+    - [Push Tokens And Multiple Profiles](#push-tokens-and-multiple-profiles)
+  - [Request Push Notification Permission](#request-push-notification-permission)
+  - [Receiving Push Notifications](#receiving-push-notifications)
+    - [Tracking Open Events](#tracking-open-events)
+    - [Rich Push](#rich-push)
+    - [Deep Linking](#deep-linking)
+      - [Option 1: URI Schemes](#option-1-uri-schemes)
+      - [Option 2: Universal Links](#option-2-universal-links)
+- [Additional Details](#additional-details)
+  - [Sandbox Support](#sandbox-support)
+  - [SDK Data Transfer](#sdk-data-transfer)
+  - [Retries](#retries)
+  - [License](#license)
+  - [UserDefaults access (SDK version < 3.0.0)](#userdefaults-access-sdk-version--300)
 
 ## Overview
 
-KlaviyoSwift is an SDK, written in Swift that can be integrated into your iOS App. The SDK enables you to engage with your customers using push notifications. In addition you will be able to take advantage of Klaviyo's identification and event tracking functionality. Once integrated, your marketing team will be able to better understand your app users' needs and send them timely messages via APNs.
+The Klaviyo Swift SDK allows developers to incorporate Klaviyo's analytics and push notification functionality in their iOS applications.
```

Comment:
nit: `The Klaviyo Swift SDK allows developers to incorporate Klaviyo's analytics and push notification functionality into their iOS applications.` 

`into their` instead of `in their` maybe?